### ISSUE TITLE
Overlap images in products

### ISSUE THREAD

**Issue by archisss** [bagisto/bagisto#4065]
# Bug report

### Title
When you are seeing and item and you scroll down the images overlap with the content 

### Issue Description
When you are seeing and item and you scroll down the images overlap with the content 

### Preconditions
**Please provide as detailed information about your environment as possible.**

    1. Demo version.

### Steps to reproduce
**It is important to provide a set of clear steps to reproduce this bug.If relevant please include code samples.**

    1. Open a new Demo version 
    2. Click on some product
    3. Scroll Down your mouse 

### Expected result
The image should gone so the client can read all the information about the article

**Comment by ricardobarantini** [comment]
@archisss The bug that you describe happen only in small devices, right?
![Screen Shot 2020-10-09 at 19 13 32](https://user-images.githubusercontent.com/1111361/95635770-b8004e80-0a63-11eb-97a1-1936faea6a54.png)

**Comment by archisss** [comment]
@ricardobarantini  
No, the screenshot i post its when i open the demo on my laptop with and external monitor connected 
because i was looking projects to contribute
